### ISSUE TITLE
Properly handle documentation without types

### ISSUE THREAD

**Issue by backus** [backus/yardcheck#14]
It is valid to write something like

```ruby
# @param foo no type specified!
# @return some value of unspecified type
def thing(foo)
end
```

but right now this result in a reported offense like so

```
+Expected App#thing to receive  for foo but observed Fixnum
Expected App#thing to return  but observed NilClass
```

I think eventually we should emit a warning for this sort of documentation because it is pretty easy for people to specify the type (or write `void` for the return or `Object` for the param). The simple initial solution though is probably just to ignore these
